3_extracting_features/3.rename_cyto_locations.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
discussion point:

Why do you need two `.sh` scripts? Why not just one?

Reply:
There needs to be two .sh files because you can not run that renaming function from the .py file until the user has run the DeepProfiler script. If we only had one file and notebook like before that ran all the functions (i.e compiling DP project and renaming cyto location files) before running DeepProfiler, then DeepPorfiler would not be able to run because the file names do not fit the expected format (e.g `{well}-{site}-Nuclei.csv`). Sp. we need to run the compiling project .sh file, then DeepProfiler script, then the renaming .sh file.